### ISSUE TITLE
Regarding CNN  

### ISSUE THREAD

**Issue by kenil22** [innovator-arjun/Solving-Mathematical-equation-Handwritten-CNN#1]
model = Sequential()
model.add(Conv2D(32, (3,3), input_shape=(28, 28,1), activation='relu',padding='same'))
model.add(MaxPooling2D((2,2), padding='same'))
model.add(Conv2D(15, (3, 3), activation='relu', padding='same'))
model.add(MaxPooling2D(pool_size=(2, 2), padding='same'))
model.add(Dropout(0.2))
model.add(Flatten())
model.add(Dense(128, activation='relu'))
model.add(Dense(50, activation='relu'))
model.add(Dense(13, activation='softmax'))

model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy'])

# Above is the Convolutional Neural Network, Below is the error shows up when we fit the model
Default MaxPoolingOp only supports NHWC on device type CPU
	 [[{{node sequential_6/max_pooling2d_6/MaxPool}}]] [Op:__inference_train_function_1816]
